### PR TITLE
INSP: support `pub extern crate` reexports

### LINKED ISSUE #3396: Reexported types does not display in autocompletion
## Environment

* **Rust toolchain version:**

[loomaclin@loomaclin ~]$ rustup show
Default host: x86_64-unknown-linux-gnu

installed toolchains

stable-x86_64-unknown-linux-gnu
nightly-2018-06-01-x86_64-unknown-linux-gnu
nightly-2018-10-14-x86_64-unknown-linux-gnu
nightly-2019-01-17-x86_64-unknown-linux-gnu
nightly-x86_64-unknown-linux-gnu (default)

installed targets for active toolchain

wasm32-unknown-unknown
x86_64-unknown-linux-gnu

active toolchain

nightly-x86_64-unknown-linux-gnu (default)
rustc 1.34.0-nightly (b244f61b7 2019-02-12)

* **IDE name and version:**
IntelliJ IDEA 2018.3.3 (Community Edition)
Build #IC-183.5153.38, built on January 9, 2019
JRE: 1.8.0_152-release-1343-b26 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.20.7-arch1-1-ARCH
* **Operating system:**
[loomaclin@loomaclin ~]$ archey3

               +                OS: Arch Linux x86_64
               #                Hostname: loomaclin
              ###               Kernel Release: 4.20.7-arch1-1-ARCH
             #####              Uptime: 1:33
             ######             WM: None
            ; #####;            DE: GNOME
           +##.#####            Packages: 990
          +##########           RAM: 6002 MB / 15741 MB
         #############;         Processor Type: Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
        ###############+        $EDITOR: None
       #######   #######        Root: 69G / 117G (58%) (ext4)

## Problem description
reproduce gif

1) `ggez` crate re-exported crate `mint`: https://docs.rs/ggez/0.5.0-rc.0/ggez/?search=#reexports
2) `mint` have public struct `Point2`: https://docs.rs/mint/0.5.1/mint/struct.Point2.html
3) but it dosnt display in autocompletion

## Steps to reproduce
1) Create empty project.
2) Add "ggez = "0.5.0-rc.0"
3) write `fn test(point: Point2<f32>)`
4) try to use autocomplete on `Point2<f32>`

Repo with reproduce: https://github.com/LooMaclin/reproduce_autoimport_reexported_types_idea_rust_plugin

### PR DESCRIPTION
**Pull request by Undin**
Fixes #3396

### CODE REVIEW

File: src/test/kotlin/org/rust/lang/core/completion/RsPathCompletionFromIndexTest.kt [hunk 1]
```diff
@@ -6,14 +6,16 @@
...
+import org.rust.WithDependencyRustProjectDescriptor
 import org.rust.hasCaretMarker
 import org.rust.ide.settings.RsCodeInsightSettings
 import org.rust.lang.core.completion.RsCommonCompletionProvider.Testmarks
 import org.rust.openapiext.Testmark
 
 class RsPathCompletionFromIndexTest : RsCompletionTestBase() {
 
-    fun `test suggest an non-imported symbol from index and add proper import`() = doTest("""
+    fun `test suggest an non-imported symbol from index and add proper import`() = doTestByText("""
```
Comment (vlad20012) [inline src/test/kotlin/org/rust/lang/core/completion/RsPathCompletionFromIndexTest.kt]: We usually use names like `...byCode`, isn't it?

Comment (Undin) [reply src/test/kotlin/org/rust/lang/core/completion/RsPathCompletionFromIndexTest.kt]: In `RsAnnotationTestBase` we use `byText` and `byFileTree` suffix, i.e. all annotator and inspection tests uses base methods with such prefixes

**Comment by Undin** [comment]
@vlad20012 ready for review

**Comment by mchernyavsky** [comment]
bors retry

**Comment by mchernyavsky** [comment]
bors r=vlad20012

**Pull request merged**